src/cards/ha-media_player-card.html

```diff
@@ -219,13 +235,75 @@
       value: 1,
       reflectToAttribute: true,
     },
+
+    playbackDuration: {
+      type: Number,
+    },
+
+    playbackPosition: {
+      type: Number,
+    },
+
+    playbackVisible: {
+      type: Boolean,
+      value: false
+    },
+
+    _internal: {
```

Comment:
If it is not going to be used inside the templates or as callbacks, no need to add it as attributes.